doc/development_process/index.rst

```diff
@@ -15,6 +15,7 @@ Development Model
    api_lifecycle
    communication
    documentation
+   gh_labels
```

Comment:
IMO this should not be a top level section, it should go under something else like issues or proposals or the dev process.